app/models/like.rb

```diff
@@ -0,0 +1,8 @@
+class Like < ApplicationRecord
+  belongs_to :author, class_name: :User
+  belongs_to :post
+
+  def update_likes_count
+    post.increment!(:likes_counter)
+  end
```

Comment:
## Optional
- [ ] I highly suggest that you use [Rails Active Callback](https://guides.rubyonrails.org/active_record_callbacks.html), `after_save` to call the `#update_likes_count` method every time a new like has been saved to the database, then make the `update_likes_count` method private